src/Stratis.Bitcoin/TxIndexing/TransactionIndexer.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DBreeze;
+using DBreeze.DataTypes;
+using DBreeze.Utils;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.EventBus;
+using Stratis.Bitcoin.EventBus.CoreEvents;
+using Stratis.Bitcoin.Primitives;
+using Stratis.Bitcoin.Signals;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.TxIndexing
+{
+    // TODO THIS CLASS IS NOT FINISHED AND NOT USED
+    public class TransactionIndexer : IDisposable
```

Comment:
@noescape00 what was the reasoning behind creating a separate class for the transaction indexing as it is already in the block store?

Reply:
Block store stores txes by ID. 
Here we will store address data by address. 

So just different datasets. 

Maybe I should rename it to AddressIndexer, since tx indexer is a bit misleading it seems.

Reply:
Ah yes, that makes more sense now. Rename this class to a more appropriate name 👍

Reply:
I very much like this component but I don't think its should live in Stratis.Bitcoin but in a feature.
As its not activated yet I don't mind pushing like this and moving it out when activating it.